pkg/generators/snippets/mergemap.go

```diff
@@ -0,0 +1,35 @@
+package snippets
+
+import (
+	"errors"
+	"io"
+
+	"k8s.io/gengo/generator"
+)
+
+func GenerateMergeMapStringString(w io.Writer, ctx *generator.Context) error {
+	if ctx == nil {
+		return errors.New("nil pointer")
+	}
+
+	sw := generator.NewSnippetWriter(w, ctx, "$", "$")
+
+	sw.Do("// mergeMapStringString creates a new map and loads it from map args\n", nil)
```

Comment:
Does it make sense to move these `sw.Do` calls to a loop over a raw text blob. [This feels much more readable to me.](https://go.dev/play/p/pToUqcWNMIr)

Reply:
Thanks, that actually makes this and the others much more readable. I'll fix.